### PR TITLE
Add RISC-V notification group instructions

### PR DESCRIPTION
**Pull request by JohnTitor**
~~This is still WIP as we don't have the Zulip stream (`#t-compiler/risc-v`) yet.~~
Now it's ready to review!

### CODE REVIEW

**Comment by JohnTitor** [comment]
Okay, it's ready to review -- @rust-lang/wg-rustc-dev-guide could someone review this?

File: src/notification-groups/risc-v.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+This list will be used to ask for help both in diagnosing and testing
+RISC-V-related issues as well as suggestions on how to resolve
+interesting questions regarding our RISC-V support.
+
+The group also has an associated Zulip stream ([`#t-compiler/risc-v`])
+where people can go to pose questions and discuss RISC-V-specific
+topics.
+
+So, if you are interested in participating, please sign up for the
+RISC-V group! To do so, you open a PR against the [rust-lang/team]
```
Comment (LeSeulArtichaut) [inline src/notification-groups/risc-v.md:16]:
```suggestion
RISC-V group! To do so, open a PR against the [rust-lang/team]
```

Comment (JohnTitor) [reply src/notification-groups/risc-v.md]: Thanks! Applied it to ARM and Windows instructions as well.

**Comment by LeSeulArtichaut** [comment]
Thanks!

**Pull request merged**